datastore/zmq/disabled.go

```diff
@@ -0,0 +1,16 @@
+//go:build !has_zmq
+
+package zmq
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/teslamotors/fleet-telemetry/metrics"
+	"github.com/teslamotors/fleet-telemetry/telemetry"
+)
+
+func NewProducer(context.Context, *Config, metrics.MetricCollector, *logrus.Logger) (telemetry.Producer, error) {
+	return nil, fmt.Errorf("Not built with ZMQ support, build with -tags has_zmq")
```

Comment:
can you update readme for how to build with tags?

Reply:
I added a small section on setting the correct GO_FLAGS for building zmq. In terms of the Dockerfile, what would you recommend? Preferrably zmq if off by default because it adds non-trivial build times

Reply:
Do you know on top of your mind how much more to build time? We do want users to pull from fleet-telemetry image from dockerhub instead of compiling the repository themselves. So I would say, we need all dependencies there

Reply:
I will run a quick test and get back to you

Reply:
Impact is low, in my tests the docker build took no more than a couple seconds longer. My view was skewed becuase I have built the lib from source in the past, which takes a while :)